### ISSUE TITLE
New Selectable Buttons field does not behave in Publish Layouts

### ISSUE THREAD

**Issue by murtaugh** [ExpressionEngine/ExpressionEngine#2012]
When you try to edit the position of a Selectable Buttons field in a Publish Layout, the new field type always defaults to the last position in the first tab. If you try and drag it elsewhere, upon saving you'll find that the position of the field above the Selectable Buttons field has changed instead.

I thought it might be due to the new field being part of a conditional setup, but I created a new test Selectable Buttons field in a separate channel and the problem persists.

This is in EE 6.3.1

**Comment by intoeetive** [comment]
@murtaugh I wasn't able to replicate this with a quick generic test - could be something specific to fields setup that you have?

I know drag & drop in layouts does not always work 100% well, so maybe you should try once again.

If the problem persists, any chance that you could record a screencast? Also, any javascript errors in the developer console?

**Comment by murtaugh** [comment]
I'm certain it's not drag and drop being annoying, I'm very familiar with how it likes to misbehave :)

Yesterday I tested it on a publish layout with a lot of fields, as well as a brand new channel with only the new field, and it was behaving the same.

I tested those two again just now, and the issues persisted, but, of course, as soon as I set up for a screencast, it started behaving, without changing anything, and I can't get it to happen again. Should have thought to screen record it yesterday I guess, but all seems well at the moment. 🤷‍♂️
